### PR TITLE
RamenPos v2.4.1

### PR DESCRIPTION
**Pull request by metinavc1**

### CODE REVIEW

File: ramenpos/build.gradle [hunk 1]
```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java'
+    id 'application'
+    id 'com.github.johnrengelman.shadow' version '1.2.4'
+}
+
+mainClassName = 'com.assemblypayments.ramenpos.Main'
+
+sourceCompatibility = 1.8
```
Comment (gouline) [inline ramenpos/build.gradle]: `JavaVersion.VERSION_1_8`

File: ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package com.assamblypayments.ramenpos;
+
+public class Enums {
```
Comment (gouline) [inline ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java]: This class contains no enums, why is it called `Enums`?

File: ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    static final String Pair = "Pair";
```
Comment (gouline) [inline ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java]: Constants in Java are named in `CAPITAL_CASE`, not camel case.

File: ramenpos/src/main/java/com/assamblypayments/ramenpos/FormAction.java [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+        formMain.options.setMerchantReceiptHeader(sanitizePrintText(txtAction1.getText().trim()));
+        formMain.options.setCustomerReceiptFooter(sanitizePrintText(txtAction2.getText().trim()));
+        formMain.options.setMerchantReceiptFooter(sanitizePrintText(txtAction2.getText().trim()));
+
+        lblFlowMessage.setText("# --> Receipt Header and Footer is entered");
+        formMain.getOKActionComponents();
+    }
+
+    private String sanitizePrintText(String printText) {
+//        printText = printText.replace("\\emphasis", "\emphasis");
```
Comment (gouline) [inline ramenpos/src/main/java/com/assamblypayments/ramenpos/FormAction.java]: No commented out code in samples.

File: settings.gradle [hunk 5]
```diff
@@ -1 +1,2 @@
 include 'acmepos', 'kebabpos', 'motelpos', 'tablepos', 'utils'
+include 'ramenpos'
```
Comment (gouline) [inline settings.gradle]: Why is it a separate statement? It should go in with the rest of them.

**Review by gouline — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 5 above)
Comment (metinavc1) [reply settings.gradle]: IDE does it automatically

File: ramenpos/build.gradle (hunk 1 above)
Comment (metinavci) [reply ramenpos/build.gradle]: changed to 1.7

Comment (metinavc1) [reply ramenpos/build.gradle]: done

File: ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java (hunk 2 above)
Comment (metinavc1) [reply ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java]: done

File: ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java (hunk 3 above)
Comment (metinavc1) [reply ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java]: Done

File: ramenpos/src/main/java/com/assamblypayments/ramenpos/FormAction.java (hunk 4 above)
Comment (metinavc1) [reply ramenpos/src/main/java/com/assamblypayments/ramenpos/FormAction.java]: Done

**Review by gouline — APPROVED** [review]

**Pull request merged**